app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert 17 == sum(split_integer(17, 4))
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    result = split_integer(40, 5)
```

Comment:
I think you do not need to create a variable for this test (check the `checklist` file for this task). You can write:
`assert split_integer(40, 5) == [8, 8, 8, 8, 8]`